src/MLJGLMInterface.jl

```diff
@@ -103,7 +106,7 @@ function split_X_offset(X, offsetcol::Symbol)
     ct = Tables.columntable(X)
     offset = Tables.getcolumn(ct, offsetcol)
     newX = Base.structdiff(ct, NamedTuple{(offsetcol,)})
-    return newX, Vector(offset)
+    return newX, _to_vector(offset)
```

Comment:
```suggestion
    return newX, collect(offset)
```
I think collecting this is fine indeed. The offset shouldn't be too big, so the added complexity of some AbstractVector or Iterable or something is not worth it per se

Reply:
Yeah, the reason I did it this way was to avoid allocation when `v` is already a `Vector`.

Reply:
Where would that make a difference? It only saves a megabyte for an offset column of 100 000 entries.

```julia
julia> @btime _to_vector(rand(100_000));
  66.651 μs (2 allocations: 781.33 KiB)

julia> @btime collect(rand(100_000));
  94.693 μs (4 allocations: 1.53 MiB)
```

Reply:
I don't seem any harm in saving an extra megabyte in this case as the function call automatically gets inlined by the compiler.

Reply:
Good arguments :+1: